cli/f3.go

```diff
@@ -183,3 +324,111 @@ func prettyPrintManifest(out io.Writer, manifest *manifest.Manifest) error {
 	}
 	return t.ExecuteTemplate(out, "manifest", manifest)
 }
+
+func prettyPrintFinalityCertificate(out io.Writer, cert *certs.FinalityCertificate, entries gpbft.PowerEntries) error {
+	if cert == nil {
+		_, _ = fmt.Fprintln(out, "Certificate: None")
+		return nil
+	}
+
+	const certificateTemplate = `---
+Instance: {{.GPBFTInstance}}
+Finalized Chain:
+  {{ .ECChain }}
```

Comment:
This doesn't pretty-print well. We should probably make it look something like `chain list`, and should probably have a sane default max. Something like:

```
Finalized Chain (epochs A-B):
  Tipset A # Head
  Tipset B
  ... (ommitted N tipsets)
```

Reply:
I went with a different format as the chain list format was a tad too complicated for F3 purposes. See example output in later comments. Please let me know what you think.